### ISSUE TITLE
TalkBack support when setFocusable(false)

### ISSUE THREAD

**Issue by alashow** [skydoves/Balloon#407]
**Describe the Bug:**

Using `setFocusable(false)` allows touching other views without having to click twice ( if `setFocusable(true)`, need to click once outside balloon to dismiss ballon and another to click for intended view)

However, `setFocusable(false)` disables TalkBack behavior when balloon is shown. Only after clicking the ballon window TalkBack service announces ballon windows text.

**Comment by skydoves** [comment]
Hi @alashow, thanks for reporting this issue. I think this issue will be fixed in #406

**Comment by skydoves** [comment]
You can available the fixed version with `1.5.2-SNAPSHOT` following the [instructions](https://github.com/skydoves/Balloon#snapshot).

**Comment by alashow** [comment]
@skydoves Seems this wasn't fixed in #406.  Setting `setFocusable` is still disabling auto announcement of tooltip window text. Tested in Android 13 on Galaxy Tab S8 using `1.5.3-SNAPSHOT` version

**Comment by alashow** [comment]
@skydoves should I open a new issue?

**Comment by skydoves** [comment]
Hey @alashow, sorry for the delayed response. Yes, if you open a new issue, I will take a look at it. Thank you!
